apps/challenges/admin.py

```diff
@@ -42,6 +42,7 @@ class ChallengeAdmin(ImportExportTimeStampedAdmin):
     list_display = (
         "id",
         "title",
+        "domain",
```

Comment:
We do not necessarily need it in the admin page. Please keep this file unchanged.